client/src/components/Indices/IndexFilter.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <span>
+        <b-input-group>
+            <DebouncedInput :delay="debounceDelay" v-slot="{ value, input }" v-model="localFilter">
```

Comment:
Looks like `value` here shadows the `value` prop on the component; rename in the destructure for clarity?

Reply:
Done, I think!